examples/step-70/doc/intro.dox

```diff
@@ -0,0 +1,488 @@
+<br>
+
+<i>This program was contributed by Luca Heltai (SISSA, Trieste) and Bruno Blais (Polytechnique Montréal)
+</i>
+
+<b>Change this!!!!</b>
```

Comment:
:-)

Reply:
This should be fixed before the release, but it can wait for the release candidate.